src/components/alert/package.json

```diff
@@ -8,12 +8,18 @@
       {
         "component": "BAlert",
         "props": [
+          {
+            "prop": "variant",
+            "settings": true,
```

Comment:
Couldn't we look this up from the global default config object?  If the key exists in the nested config object, then it would be set to true in the props items array?  Would save us from having to keep a flag in the meta data.... as not all prop descriptions are not in the component meta file (some come from the common props)

We import the default config into the docs (for the misc section).

Reply:
Maybe as a check, we look for the prop in the default config, and then check the options to see if the prop default is a function... then we flag it as being settings?

Reply:
I thought about that too but maybe the name in the settings differs from the property name at some time.

Reply:
Hmmmm yeah....  but it should be the same `camelCaseName` as the prop...

Reply:
How many oddballs do we have?